portal/code/company-portal/src/stores/appstate.ts

```diff
@@ -166,6 +166,7 @@ const L = {
   companyName: 'DMG MORI', url: ''
 };
 
+
```

Comment:
Could you please ignore this file ?  somehow whitespaces detected as changes here

Reply:
You probably added a new line.

Reply:
most probably :) it didn't effect anything though. Do you think this PR is able to merge ? @vsamarora @Siegfriedk